app/views/carts/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if @books.present? %>
+	<%= @books.each do |book| %>
```

Comment:
Should this be
```suggestion
	<%- @books.each do |book| %>
```

Reply:
I think `<% @books.each do |book| %>` is also ok. I read this document: [https://api.rubyonrails.org/classes/ActionView/Base.html](url), seem `<% %>` `<%- -%>` is same. What do you think about it?

Reply:
I'm fine with either, but we should not use `<%=`